src/components/ClaimForm.jsx

```diff
@@ -0,0 +1,69 @@
+import { useParams } from "react-router-dom";
+import H3 from "./atoms/H3";
+import P from "./atoms/P";
+import Highlighted from "./atoms/Highlighted";
+import { useState } from "react";
+
+
+export default function ClaimForm ({person, getBookDetails}) {
+
+    const {id} = useParams()
+    const [nameError, setNameError] = useState("")
+    const [emailError, setEmailError] = useState("")
+    const [name, setName] = useState("")
+    const [email, setEmail] = useState("")
+
+
+    const data = {
+        "name":name,
+        "email":email,
+    };
+
+    const requestOptions = {
+        method: "PUT",
+        mode: 'cors',
+        headers: { "Content-Type": "application/json",
+                    "Accept": "application/json"
+                },
+        body: JSON.stringify(data)
+    }
```

Comment:
You should move these two consts into sendClaimForm, since they aren't needed anywhere outside of it.

Literally just move them in above the fetch request